tests/resources/test_resources_review.py

```diff
@@ -224,3 +224,233 @@ def test_delete_no_review(
     # Try to delete an open review - should not be possible
     req = client.delete(review_link, headers=headers)
     assert req.status_code == 400
+
+
+def test_submission_closed_non_member(
+    client,
+    minimal_record,
+    closed_submission_community,
+    headers,
+    uploader,
+):
+    """Test different users ability ro submit record with closed submission policy."""
+    client = uploader.login(client)
+
+    # Request a review
+    minimal_record["parent"] = {
+        "review": {
+            "type": CommunitySubmission.type_id,
+            "receiver": {"community": closed_submission_community.data["id"]},
+        }
+    }
+
+    # Create a draft
+    draft = client.post("/records", headers=headers, data=json.dumps(minimal_record))
+    links = draft.json["links"]
+    review = draft.json["parent"]["review"]
+    assert draft.status_code == 201
+    assert "submit-review" in links
+    assert "id" in review
+    assert review["receiver"] == {"community": closed_submission_community.data["id"]}
+    assert review["type"] == CommunitySubmission.type_id
+    assert draft.json["parent"]["communities"] == {}
+
+    # Submit for review
+    # # as a non-member
+    response = client.post(link(links["submit-review"]), json={}, headers=headers)
+    assert response.status_code == 403
+    assert (
+        response.json["message"]
+        == "Submission to this community is only allowed to community members."
+    )
+
+    # # as an unauthorized user
+    client = uploader.logout(client)
+    response = client.post(link(links["submit-review"]), json={}, headers=headers)
+    assert response.status_code == 403
+    assert response.json["message"] == "Permission denied."
+
+
+def test_submission_closed_member(
+    client,
+    minimal_record,
+    closed_submission_community,
+    headers,
+    community_owner,
+):
+    """Test different users ability ro submit record with closed submission policy."""
```

Comment:
```suggestion
    """Test different users ability to submit record with closed submission policy."""
```

maybe a bit more descriptive docstring for the specific case this tests validates so we don't rely on the test name?